### PR TITLE
Feature/find mmg private headers

### PR DESCRIPTION
**Pull request by Algiane**
## Find Mmg private headers

Update ParMmg to find Mmg headers after commit `f3c20cb1b8e638208880efc9` of Mmg.
Linked to:
  - PR https://github.com/MmgTools/mmg/compare/develop...feature/mmg-install-private-headers
  - issue #31
  
  It also include some of the propositions of improvement of @prudhomm to use ParMmg as submodule (not all of them for now).

### CODE REVIEW

File: src/libparmmgtypes.h [hunk 1]
```diff
@@ -32,7 +32,7 @@
 #ifndef _LIBPARMMGTYPES_H
 #define _LIBPARMMGTYPES_H
 
-#include "mmg/mmg3d/libmmgtypes.h"
+#include "libmmgtypes.h"
```
Comment (prj-) [inline src/libparmmgtypes.h:35]:
```suggestion
#include "mmg/common/libmmgtypes.h"
```
That's the single diff I have to apply to make everything work, last I checked.

**Comment by Algiane** [comment]
Thanks a lot for your responsiveness and for your help. As the last issue is in ParMmg and not  Mmg I will merge the current PR and solve the issue of header inclusion in ParMmg later (that I still don't understand because `mmg/common` is added to ParMmg include directories).

Best Regards,

**Comment by prj-** [comment]
No problem, we can iterate once the Mmg branch is merged. Worst case scenario, you can't wrap things up on the ParMmg side, then I'll just need to keep the ParMmg fork. At least I'll be able to redirect the trafic on the official Mmg repository.

**Comment by Algiane** [comment]
PR will be merged to follow the merge of the PR https://github.com/MmgTools/mmg/pull/187 of Mmg but we may still have some header inclusions to fix in ParMmg.

**Pull request merged**